Sources/SwiftFusion/Core/Dictionary+Differentiable.swift

```diff
@@ -16,7 +16,7 @@ extension Dictionary: Differentiable where Value: Differentiable {
   }
 
   public var zeroTangentVectorInitializer: () -> TangentVector {
-    { mapValues { _ in .zero } }
+    { mapValues { v in v.zeroTangentVectorInitializer() } }
```

Comment:
`Differentiable` has a default computed field `.zeroTangentVector` that returns `.zeroTangentVectorInitializer()`.

Reply:
And you suggest I use that instead?

Reply:
Yes, I think it's easier to understand. But if you disagree then I'm fine keeping it as `.zeroTangentVectorInitializer()`.

Reply:
OK, I just pushed a change that replaces all of those.  Arguably this is a 3.2% speedup (go figure)